modules/drivers/lidar/lidar_robosense/include/parser/robosense_status.h

```diff
@@ -0,0 +1,93 @@
+/******************************************************************************
+ * Copyright 2021 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+
+#pragma once
+#include <memory>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "cyber/cyber.h"
+#include "modules/drivers/lidar/lidar_robosense/proto/sensor_suteng.pb.h"
+#include "modules/drivers/lidar/lidar_robosense/proto/sensor_suteng_conf.pb.h"
+#include "modules/drivers/proto/pointcloud.pb.h"
+
+namespace apollo {
+namespace drivers {
+namespace robosense {
+
+constexpr double RADIANS_TO_DEGREES = 180.0 / 3.1415926535897;
+constexpr double DEGRESS_TO_RADIANS = 3.1415926535897 / 180.0;
```

Comment:
Check wether DEG_TO_RAD has been defined elsewhere in Apollo. Avoid hard-code PI value. Use the one from `<cmath>` instead.